test/unit_tests/apps/youth-employment-success/js/views/goals-spec.js

```diff
@@ -97,4 +97,18 @@ describe( 'goalsView', () => {
       updateGoalTimelineAction( shortTimelineEl.value )
     );
   } );
+
+  it( 'preserves spaces in the textarea', () => {
+    const ltgEl = document.querySelector( `.${ CLASSES.LONG_TERM_GOAL }` );
+    const goal = 'This is my goal\nIt is a good goal\nThe end!';
```

Comment:
Story of my life

Reply:
At first I was going to go all the way and repurpose the rifle lines from full metal jacket, but it seemed a little too dark for a PR 😄